Client/Frontend/Reader/ReaderMode.js

```diff
@@ -119,6 +119,23 @@ var _firefox_ReaderMode = {
             }
         }
     },
+
+    // Loop over the style sheets, find all the the @font-face declarations and fix the
+    // relative src attributes so that they point to our embedded web server.
+    fixFontFaces: function(webServerBase) {
```

Comment:
We start the server with port `0` so that the OS will allocate it a random available port. If we use a hardcoded port number then we have to deal with the possibility that the port is not available and that we have to use a different port. And then we are back to dynamically changing the port in things like this.